frontend/main.js

```diff
@@ -175,6 +205,13 @@ async function startApp () {
     }
   })
 
+  sbp('chelonia/externalStateSetup', { reactiveSet: Vue.set, reactiveDel: Vue.delete })
+
+  /* sbp('okTurtles.events/on', NAMESPACE_REGISTRATION, ({ name, value }) => {
+    const cache = sbp('state/vuex/state').namespaceLookups
+    Vue.set(cache, name, value)
+  }) */
```

Comment:
Is this code important? If so, it needs to be uncommented and placed in the right file. If not, should be cleaned up / removed.

Reply:
Added comment above  it